### ISSUE TITLE
Provide both toggle states count

### ISSUE THREAD

**Issue by vvo** [algolia/instantsearch.js#919]
At the moment, we only provide the count for the current state of the toggle. A good idea from @davidpike7 is to also provide the count for the other state of the toggle.

So that you can show things like "Only show free shipping (20)" 20 being the number of free shipping items. It's easier to understand than "Only show free shpping (340)" Where 340 is the actual number of items not having free shipping.

**Comment by vvo** [comment]
We need a POC before going further, unsure all the API needs are here in the helper

**Comment by vvo** [comment]
Hi @davidpike7, I just did some more fixes on the toggle widget, can you this build on your website: https://gist.githubusercontent.com/vvo/19d16de5d738ab55eaa3a9e1e3c7b67b/raw/71ff01360b6591186142b1457d5add5f40f746a3/instantsearch.min.js

Here's what's new:

```text
    feat(toggle): Provide a better default widget
    
    This update brings:
    - default to showing the next user action items count instead of the
    current number of displayed hits
    
    which means:
    if checkbox is not checked, show: [ ] free shipping (countWhenChecked)
    if checkbox is checked, show: [x] free shipping (countWhenNotChecked)
    - fix numeric based on and off values (fixes #1096)
    - provide `onFacetValue` and `offFacetValue` template data, fixes #919
```
